content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -71,6 +71,7 @@ The available AccountSet flags are:
 | Flag Name          | Decimal Value | Corresponding Ledger Flag | Description   |
 |:-------------------|:--------------|:--------------------------|:--------------|
 | `asfAccountTxnID`  | 5             | (None)                    | Track the ID of this account's most recent transaction. Required for [`AccountTxnID`](transaction-common-fields.html#accounttxnid) |
+| `asfAuthorizedMinter` :not_enabled:| 10         | (None)       | Enable to allow an account to mint tokens on behalf of another account. Specify the authorized account in the `minter` field of the [AccountRoot](accountroot.html) object. |
```

Comment:
Some other notes about this field:

- I think I've convinced the team to remove this flag from the implementation since it is an unnecessary complication. So the final version probably won't have this at all, but it's unknown how long until that change makes it to the NFT-Devnet.
- `Minter` is case-sensitive and needs to be capitalized
- The description could better contextualize who's authorizing whom. Specifically, "Allow another account to mint non-fungible tokens on this account's behalf." (Also: say NFTs rather than just "tokens" since this does not apply to fungible tokens.)